test/api/Routes.spec.js

```diff
@@ -0,0 +1,40 @@
+import { expect } from 'chai';
```

Comment:
Does the test exit properly or do you have to ctrl+c after it runs?

Reply:
Yes it exits properly.

Reply:
Although be aware: since the test spins up the app, the logger and test print to the same console, so it looks like this (with logger logs and test information kinda mixed up)...

<img width="587" alt="screen shot 2018-05-01 at 10 30 46 am" src="https://user-images.githubusercontent.com/3158226/39476700-e7952b46-4d2a-11e8-96b3-eb2d38ff4e7d.png">